projects/Mallard/src/helpers/fetch.ts

```diff
@@ -189,6 +190,30 @@ const fetchWeather = <T>(
     )
 }
 
+export const fetchCacheClear = async (): Promise<boolean> => {
+    try {
+        const response = await fetch(defaultSettings.cacheClearUrl)
+        const cacheNumber = await response.json()
+        const cacheNumberStorage = await cacheClearCache.get()
```

Comment:
Tiny little speeder-upper, but cacheNumber and cacheNumberStorage can probable go in Promise.all?